### PR TITLE
DRUP-802: #ready - A ton of updates for Bookplates, see below:

### PR DESCRIPTION
**Pull request by z3cka**
Squashed commit of the following:

commit 910f4ec0ee51e5fc8b085e14e091a32cf7c110fc
Author: Casey Grzecka <cgrzecka@library.ucla.edu>
Date:   Wed Mar 2 23:58:08 2016 +0000

    adds conditional for field in certain view and modifies field label on bookplates landing

commit acbc0d7b15c33195ba9eea545bfdf4922074a13a
Author: Casey Grzecka <cgrzecka@library.ucla.edu>
Date:   Wed Mar 2 19:59:55 2016 +0000

    removes 'and' seperator for single donor in field display

commit e02ee94562e858972d5387b454b15339056e4776
Author: Casey Grzecka <cgrzecka@library.ucla.edu>
Date:   Wed Mar 2 19:03:19 2016 +0000

    makes field_donor_name_bookplates field's Display type an unordered list and removes the simple seperator ' and ' string

commit a52da0ea432df86b80f5520c926f391591763616
Author: Casey Grzecka <cgrzecka@library.ucla.edu>
Date:   Wed Mar 2 18:22:15 2016 +0000

    appends the last donor with an 'and' before it

commit f1806809afaf3f63eadf5992773a4c699b32ff1c
Author: Casey Grzecka <cgrzecka@library.ucla.edu>
Date:   Wed Mar 2 01:17:07 2016 +0000

    WIP on adding commas between the donors. TODO: insert 'and' before the last donor sans oxford comma

commit cbba53e4eeaa7c060ef989655c63dd226661e4c3
Author: Casey Grzecka <cgrzecka@library.ucla.edu>
Date:   Tue Mar 1 23:34:51 2016 +0000

    adds conditional that hides uclalib_bookplates_spak_args field when empty

commit ab29ebe551de2de763df1433878c270d9473be32
Author: Elaine Blakeman <elaineblakeman@library.ucla.edu>
Date:   Fri Feb 26 12:46:59 2016 -0800

    DRUP-802: #ready adds view for term variant

commit 4074873edaa2270a11572fce1b766b688245be3e
Author: Casey Grzecka <cgrzecka@library.ucla.edu>
Date:   Wed Feb 24 20:11:24 2016 +0000

    adds target attribute to field_bookplates_spak_code field preprocessor to open link in a new browser window

commit b870b6af5aeff2b6779a9e382182ccabe554eb71
Author: Casey Grzecka <cgrzecka@library.ucla.edu>
Date:   Wed Feb 24 19:31:47 2016 +0000

    removes rewrite results in field handler for field_bookplates_spak_code in views

commit 23f19141883347894701c02af508cf3578b7002c
Author: Casey Grzecka <cgrzecka@library.ucla.edu>
Date:   Wed Feb 24 19:16:29 2016 +0000

    removes broken handler from bookplates view

commit ba10542dda2c5bff42c75ccc6ff2cf2f09cba1f9
Author: Casey Grzecka <cgrzecka@library.ucla.edu>
Date:   Wed Feb 24 19:08:16 2016 +0000

    removes fields that were deployed to www-test and are no longer needed

commit cd0697a095cbea2dc2f39029e21813f77aa75f60
Author: Casey Grzecka <cgrzecka@library.ucla.edu>
Date:   Wed Feb 24 17:58:11 2016 +0000

    adds preprocessor for field_bookplates_spak_code URL and some cole cleanup

commit fa4547fd2da3969c439d2391a3be0f69d9763481
Author: Elaine Blakeman <elaineblakeman@library.ucla.edu>
Date:   Mon Feb 22 09:11:55 2016 -0800

    Squashed commit of the following:

    commit c1dc3c98635bb2af7e8bd81e8d0891ed79a481a8
    Author: Elaine Blakeman <elaineblakeman@library.ucla.edu>
    Date:   Mon Feb 22 09:04:50 2016 -0800

        removes field_permssions module

    commit 3ffcb32e2f322740730d003d338f62d97baee09f
    Author: Elaine Blakeman <elaineblakeman@library.ucla.edu>
    Date:   Mon Feb 22 08:53:21 2016 -0800

        removing conflict markup

    commit f84b896cbfedb3cea75552aca2368fcb9b47da22
    Author: Elaine Blakeman <elaineblakeman@library.ucla.edu>
    Date:   Tue Feb 16 14:02:41 2016 -0800

        adds bookplate admin form and fixes conflict

        Conflicts:
        	www/sites/all/modules/features/uclalib_bookplates/uclalib_bookplates.module

    commit c1fc222a58e4e6297ce4551677bf4e02bb6398b0
    Author: Elaine Blakeman <elaineblakeman@library.ucla.edu>
    Date:   Fri Feb 19 15:20:32 2016 -0800

        #ready DRUP-802: deleted spak code before after fields

    commit 0e99b80aadd5a507e5c6f5c57b4e429d6ad52dba
    Author: Elaine Blakeman <elaineblakeman@library.ucla.edu>
    Date:   Tue Feb 16 08:18:10 2016 -0800

        #ready DRUP-802: all changes here: http://ucla.in/1QjFJMy

### CODE REVIEW

File: www/sites/all/modules/features/uclalib_bookplates/templates/node--bookplates.tpl.php [hunk 1]
```diff
@@ -78,8 +78,37 @@
  * @see template_process()
  */
 ?>
-<article<?php print $attributes; ?>>
-    <?php if (!empty($title_prefix) || !empty($title_suffix) || !$page): ?>
+<?php //dpm($variables); ?>
```
Comment (z3cka) [inline www/sites/all/modules/features/uclalib_bookplates/templates/node--bookplates.tpl.php]: this snuck in here, but we can remove this during the move to prod... not a `www-test` blocker.

File: www/sites/all/modules/features/uclalib_bookplates/uclalib_bookplates.pages_default.inc [hunk 2]
```diff
@@ -62,7 +62,7 @@ function uclalib_bookplates_default_page_manager_handlers() {
     ),
   );
   $display->cache = array();
-  $display->title = 'Collection Endowment Initiative';
+  $display->title = 'Collection Endowments';
```
Comment (z3cka) [inline www/sites/all/modules/features/uclalib_bookplates/uclalib_bookplates.pages_default.inc]: @emb03 is this right?

File: www/sites/all/modules/features/uclalib_bookplates/uclalib_bookplates.views_default.inc [hunk 3]
```diff
@@ -36,13 +37,13 @@ function uclalib_bookplates_views_default_views() {
   $handler->display->display_options['empty']['area']['field'] = 'area';
   $handler->display->display_options['empty']['area']['empty'] = TRUE;
   $handler->display->display_options['empty']['area']['format'] = 'full_html';
-  /* Field: Indexed Content: Bookplate Author First Name */
+  /* Field: Broken/missing handler */
   $handler->display->display_options['fields']['field_bookplate_author_first_nam']['id'] = 'field_bookplate_author_first_nam';
   $handler->display->display_options['fields']['field_bookplate_author_first_nam']['table'] = 'search_api_index_bookplates';
   $handler->display->display_options['fields']['field_bookplate_author_first_nam']['field'] = 'field_bookplate_author_first_nam';
   $handler->display->display_options['fields']['field_bookplate_author_first_nam']['label'] = '';
   $handler->display->display_options['fields']['field_bookplate_author_first_nam']['element_label_colon'] = FALSE;
```
Comment (z3cka) [inline www/sites/all/modules/features/uclalib_bookplates/uclalib_bookplates.views_default.inc]: hmmm, looks like some old handlers are still there for fields that got deleted... not a blocker, but should probably be removed.

File: www/sites/all/themes/uclalib_omega/templates/uclalib_bookplates/field--field_donor_name_bookplates.tpl.php [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+  <div style="display:inline;" class="field-item <?php print $delta % 2 ? 'odd' : 'even'; ?>"<?php print $item_attributes[$delta]; ?>>
+    <?php
+    print render($item);
+    // Add 'and' if not last item
+    if ($delta < (count($items) - 1)) {
+      print ' and ';
+    }
+    ?>
+</div>
+<?php endforeach; ?>
```
Comment (z3cka) [inline www/sites/all/themes/uclalib_omega/templates/uclalib_bookplates/field--field_donor_name_bookplates.tpl.php]: oops, meant to remove this file...

**Comment by z3cka** [comment]
found a few minor issues with this PR. Will fix and re-do for monday... deploying #203 & #204 to `www-test` now.

File: www/sites/all/modules/features/uclalib_bookplates/uclalib_bookplates.pages_default.inc (hunk 2 above)
Comment (z3cka) [inline www/sites/all/modules/features/uclalib_bookplates/uclalib_bookplates.pages_default.inc]: she said yes in slack.

Comment (emb03) [inline www/sites/all/modules/features/uclalib_bookplates/uclalib_bookplates.pages_default.inc]: Yes it's right, thanks!

**Pull request closed without merging**